_pages/open-source-day-2021-summer-contribution.md

```diff
@@ -35,6 +37,8 @@ Additionally, we have a "Choose your own adventure" option, where you have the o
 
 | Project | Description | Maintainer(s) |
 | ------- | ----------- | ------------- |
+| [Accessibility Snippets](https://github.com/intuit/accessibility-snippets) | VSCode Snippets created to help developers write accessible code. | [Kendall Gassner](https://github.com/kendallgassner) |
+| [Awesome Portfolio Websites](https://github.com/smaranjitghose/awesome-portfolio-websites) | TBA | TBA |
```

Comment:
We are waiting for Awesome Portfolio Website to get back to us on the info as they asked us to reach out before posting their info